wAbout/Logging-In.md

```diff
@@ -1,42 +1,88 @@
 ---
 title: Logging In
 layout: custom
-keywords: [login, ribbon]
-description: Logging in is simple and fast! After logging in once, you can set up auto-login, which saves you from entering credentials each time. You can also log into multiple accounts. 
+keywords: [login, ribbon, logout, new login, credentials]
+description: Logging in is simple and fast! Interject will remember your login and sign you in automatically subsequent visits.You can also add multiple logins and switch accounts easily.
 ---
 
-##  Before Using Interject, Here's How to Login 
+## Overview 
 
-Logging in is simple and fast! After logging in once, you can set up auto-login, which saves you from entering credentials each time. You can also log into multiple accounts. 
+Logging in is simple and fast! Interject will remember your login and sign you in automatically subsequent visits.You can also add multiple logins and switch accounts easily.
```

Comment:
I think the text should be changed here. The add-in doesn't actually automatically sign users in on subsequent visits. It's probably better to describe this as: "Interject will remember your login for 30 days." instead of "Interject will remember your login and sign you in automatically subsequent visits."

Reply:
Got it, I'll change that.